bitnami/kafka/templates/statefulset.yaml

```diff
@@ -94,20 +94,19 @@ spec:
           command:
             - /bin/bash
           args:
-            - -ec
+            - -cx
```

Comment:
Sorry, I missed this one.
Could you set it back to `ec`, `cx` is more for debugging purposes.

Reply:
👍 Done